playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -110,8 +110,8 @@
   # are modified to use the correct image tag.  However, this can trigger a
   # docker restart if new configuration is laid down which would immediately
   # pull the latest image and defeat the purpose of these tasks.
-  - openshift_cli
-  pre_tasks:
+  - openshift_docker
```

Comment:
This is where things are getting very gnarly, we have a pre-upgrade step that figures out what new version to use for the container image, that makes perfect sense, but we can't do that if we don't have this new registry in /etc/sysconfig/docker.